app/views/citizens/transactions/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<%= page_template page_title: t("transaction_types.page_titles.#{@transaction_type.name}"), template: :default do %>
+
+  <h2 class="govuk-heading-m"><%= t '.sub_heading' %></h2>
+
+  <div class="govuk-!-padding-bottom-4"></div>
+
+  <%= form_with(url: citizens_transactions_path, method: :patch, local: true) do |form| %>
+
+    <%= form.hidden_field :transaction_type, value: @transaction_type.name %>
+
+    <table class="govuk-table">
+      <caption class="govuk-table__caption"><%= t('date.date_period', from: date_from, to: date_to) if @bank_transactions.any? %></caption>
```

Comment:
If there are no bank transactions this will leave an empty caption tag. That may be OK, but I'd suggest this:
```erb
<%= content_tag(:caption, t('date.date_period', from: date_from, to: date_to), class: 'govuk-table__caption') if @bank_transactions.any? %>
```
which will only insert a caption tag if there are bank transactions